FirebaseAuth/Tests/Sample/Podfile

```diff
@@ -15,7 +15,7 @@ target 'AuthSample' do
   pod 'FirebaseCoreInternal', :path => '../../../'
   pod 'FirebaseAuth', :path => '../../../', :testspecs => ['unit']
   pod 'FirebaseInstallations', :path => '../../..'
-  pod 'FBSDKLoginKit'
+  pod 'FBSDKLoginKit', '< 13'
```

Comment:
CI still failed after the prior commit so there may have been some behavior changes.. 

I'm going to try just pinning the version of FBSDK used.

Reply:
Ok, this code change fixed CI: https://github.com/firebase/firebase-ios-sdk/actions/runs/3100850520/jobs/5021585987